releases/README.md

```diff
@@ -1,5 +1,14 @@
 ## containerd release process
 
+### containerd CI signal
```

Comment:
Similar comment to the above on whether it makes sense to call this just "containerd CI" versus something about Kubernetes+CRI integration testing.

Reply:
Here, we are mentioning both kubernetes CRI tests as well as the containerd github actions, in the CI signal. 
Also, this is for the release criteria, not the actual CI/Nightly jobs.

Reply:
About this comment, since its the release criteria, IMHO we can keep it as containerd CI signal instead of having k8s-CRI.